docs/products/kafka/kafka-connect/howto/s3-sink-connector-confluent.rst

```diff
@@ -0,0 +1,121 @@
+Create an S3 sink connector by Confluent
+========================================
+
+The Apache Kafka Connect® S3 sink connector by Aiven enables you to move data from an Aiven for Apache Kafka® cluster to Amazon S3 for long term storage.
```

Comment:
a minor point - I'd make the names in bold, so that it's a bit easier to read (although, the sentence is short and nice, so not a must). If you decide to change - there are several other places where we use the same.
```suggestion
The **Apache Kafka Connect® S3 sink connector by Aiven** enables you to move data from an **Aiven for Apache Kafka®** cluster to **Amazon S3** for long term storage.
```